packages/amplify_core/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 1.6.0
+
+### Features
+- feat: rename sendUserAttributeVerificationCode ([#3759](https://github.com/aws-amplify/amplify-flutter/pull/3759))
+
+### Fixes
+- fix(aft): Small fixes ([#3838](https://github.com/aws-amplify/amplify-flutter/pull/3838))
```

Comment:
*todo:* Let's exclude this. It isn't a customer facing change. aft is an internal tool. The commit modified a file in amplify_auth_cognito_dart, but it was only in the /tool dir. That change shouldn't be customer facing either.